README.md

```diff
@@ -122,6 +125,25 @@ The official documentation of the upstream layer can be found [here](https://git
 
     For more information about available context propagators, see the [Propagator settings](https://github.com/signalfx/splunk-otel-python/blob/main/docs/advanced-config.md#trace-propagation-configuration) for the Splunk distribution of OpenTelemetry Python.
 
+### Node.js wrapper
+
+The Node.js wrapper is based on Splunk Distribution of OpenTelemetry for Node.js version `0.14.0`. 
+
+The official documentation of the base distribution can be found [here](https://github.com/signalfx/splunk-otel-js).
```

Comment:
We must call those docs something else then—they're not the "official" Splunk docs. How about just "documentation"?

```suggestion
The documentation of the base distribution can be found [here](https://github.com/signalfx/splunk-otel-js).
```

Reply:
done